proto/process/agent.proto

```diff
@@ -450,6 +450,7 @@ message Connection {
 	uint32 dnsTimeouts = 27;
 	uint64 dnsSuccessLatencySum = 28;
 	uint64 dnsFailureLatencySum = 29;
+	map<uint32, uint32> dnsCountByRcode = 33;
```

Comment:
this key here is uint32, but in the system-probe PR we're representing rcodes as uint8.

Reply:
Right. The codes are 8 bits but `uint32` is the smallest fixed sized `uint` available in Protobuf.

Reply:
For example, ports are 16 bits but we use `int32` for them in protobuf.

Reply:
Got it, I didn't realize 32 was the smallest size. Thanks!